src/lib/math/mp/mp_asmi.h

```diff
@@ -749,6 +749,77 @@ inline constexpr void word3_muladd_2(W* w2, W* w1, W* w0, W x, W y) {
    *w2 = word_add(*w2, top, &carry);
 }
 
+#if defined(__BITINT_MAXWIDTH__) && (__BITINT_MAXWIDTH__ >= 3 * 64)
+template <WordType W>
+class word3 {
+   public:
```

Comment:
Perhaps add a short rationale doxygen string for future-us to remember why this has a special case?

Reply:
:+1: